### PR TITLE
Api/update specs

### PR DESCRIPTION
**Pull request by jstirnaman**
- improve task endpoint docs
- Tasks can be used with invokable scripts.
- Reset docs-release/oss branch to head of master due to changes for
  tasks. Doesn't appear to be anything new in OSS that would break.
- Removes redundant tag groups in compat spec.

Deployed to staging: https://test2.docs.influxdata.com/influxdb/cloud/api/#operation/PostTasks

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

**Comment by jstirnaman** [comment]
@evinaku I added a note in the `400` description and an example response. Waiting for docs team approval to deploy.

image

**Review by sunbryely-influxdata — APPROVED** [review]

**Review by sanderson — APPROVED** [review]
👌🏻 lgtm

**Pull request merged**
